### ISSUE TITLE
feat: disable the submit button when the form has errors

### ISSUE THREAD

**Issue by ayoayco** [astro-reactive/astro-reactive#299]

**Comment by Namatuzio** [comment]
Hi, could you assign this to me? I'd love to take a look at it. Thanks!

**Comment by ayoayco** [comment]
@Namatuzio thanks! let me know if you have questions!

**Comment by Namatuzio** [comment]
@ayoayco thanks for assigning me :) I do have a question though, from what I understand, the submit functionality is handled in `Form.astro` and is split up into its own data type in `control.types.ts` correct? I'm just confirming because I believe these are the 2 files I have to focus my attention on based on what needs to be done and my understanding of their purpose. Thanks again!

**Comment by ayoayco** [comment]
@Namatuzio yeah sounds right, but you have to do a bit of exploring on the submit control type & state and how it is added in the form because it’s unique.
